spec/spec_helper.rb

```diff
@@ -11,11 +11,12 @@
 
 class StaticProduct < ActiveRecord::Base
   self.table_name = "products"
+  jsonb_accessor :options, title: :string
```

Comment:
I'd like to keep any calls to `jsonb_accessor` out of `StaticProduct` since `StaticProduct` mostly exists for performance testing to make sure that any initializing that `jsonb_accessor` does isn't too slow.